updater/index.php

```diff
@@ -43,7 +43,7 @@ function print_patch_row($patch_row, $row_id, $enable_radiotton)
 		<td><?php if (!isset($patch_row['status'])) echo _AC("not_installed"); else echo htmlspecialchars($patch_row["status"]); ?></td>
 		<td><?php echo htmlspecialchars($patch_row["available_to"]); ?></td>
 		<td><?php echo htmlspecialchars($patch_row["author"]); ?></td>
-		<td><?php if (isset($patch_row['status'])) echo ($patch_row["installed_date"]=='0000-00-00 00:00:00')?_AC('na'):htmlspecialchars($patch_row["installed_date"]); ?></td>
+		<td><?php if (isset($patch_row['status'])) echo ($patch_row["installed_date"] == NULL)?_AC('na'):htmlspecialchars($patch_row["installed_date"]); ?></td>

```

Comment:
Would be better to use [`is_null()`](http://php.net/manual/en/function.is-null.php).